test/ExecutionFramework.h

```diff
@@ -68,6 +68,15 @@ class ExecutionFramework
 		std::map<std::string, Address> const& _libraryAddresses = std::map<std::string, Address>()
 	) = 0;
 
+	virtual bytes const& multiSourceCompileAndRunWithoutCheck(
```

Comment:
Shouldn't this be called from the regular "compileAndRun" function with just a single entry in the `_sourceCodeMap`?

Reply:
I meant that there should only be one function to implement in derived classes and this is the multi-source variant. Maybe we can even just change the signature of `compileAndRunWithoutCheck` instead? Execution frameworks / compilers that do not support multi-source inputs can just assert that there is only one file.

Reply:
Ah I see, I changed it so that `compileAndRunWithoutCheck` defaults to a multi source version.

> Execution frameworks / compilers that do not support multi-source inputs can just assert that there is only one file.

Since both syntax and semantic tests now support multi-source inputs, I think this won't be necessary.